odoo/tools/mail.py

```diff
@@ -615,13 +615,14 @@ def formataddr(pair, charset='utf-8'):
     return f"{local}@{domain}"
 
 
-def encapsulate_email(old_email, new_email):
+def encapsulate_email(old_email, new_email, include_email=False):
```

Comment:
Could be kwonly.

Not sure about the name, but I don't really have a better one so far.

Reply:
is this parameter really useful ? (`include_email`) do we still want to include the email somewhere in the code ?

Reply:
removed it, indeed we don't need to keep it